arch/risc-v/src/common/riscv_common_memorymap.h

```diff
@@ -67,7 +67,8 @@ EXTERN uintptr_t g_idle_topstack;
 
 /* Address of per-cpu idle stack base */
 
-EXTERN const uint8_t *g_cpux_idlestack[CONFIG_SMP_NCPUS];
+#define g_cpux_idlestack(cpuid) \
+   ((uintptr_t)_ebss + SMP_STACK_SIZE * (cpuid))
```

Comment:
```
(g_idle_topstack - SMP_STACK_SIZE * ((cpuid) - 1))
```

Reply:
> Just curious. Why using ebss is not good? To prevent potential write on the ram after `g_idle_topstack`?

If g_idle_topstack is removed from code base, it's fine to reference ebss directly.

> 
> > ```
> > (g_idle_topstack - SMP_STACK_SIZE * ((cpuid) - 1))
> > ```
> 
> Why is `-1`? I think it should be `+1`. `g_idle_topstack` is the stack top, not base.

Yes, you are right.